returnn/tf/layers/basic.py

```diff
@@ -6942,10 +6943,11 @@ def get_out_data_from_opts(cls, eval_locals=None, n_out=NotSpecified, sources=()
         allow_broadcast_all_sources = True
       out_type_.update(
         Data.get_common_data(
-          [s.output for s in sources], allow_broadcast_all_sources=allow_broadcast_all_sources).get_kwargs())
+          [s.output for s in sources],
+          allow_broadcast_all_sources=allow_broadcast_all_sources).get_kwargs(),
+          name="%s_output" % kwargs["name"])
```

Comment:
This indent is wrong, and also `name` should not be an argument to `update`, but to `get_common_data` instead, right?

Reply:
Urgh yeah I was so confused ... thanks!